web_demo/index.html

```diff
@@ -50,6 +50,8 @@
             position: absolute;
             top: 0%;
             left: 50%;
+            width: 100%;
+            height: 100%;
```

Comment:
Is this now required? Then please add a migration guide to the PR description!

Reply:
Not required, but by default a canvas element is very small (300x150px). It's now entirely up to the user how they decide to display it, so any `width`/`height` is fine.

Reply:
I'm not really sure where to add a migration guide, because there's nothing to migrate over here yet, because it is still not possible to set `width`/`height` without having it overriden by `egui`. The IME handling still sets the canvas style, but it'd be better to leave that to its own PR. This one is just to get the canvas resize working properly.

Reply:
So a user that has copy-pasted the old index.html would not need to change anything when updating eframe?

Reply:
They would, but it doesn't yet to the right thing. I guess what I'm asking is does the migration guide go into this PR or the followup that fixes IME?

Reply:
I guess it can wait 👍